plugin.video.embycon/resources/lib/skin_clone.py

```diff
@@ -0,0 +1,133 @@
+
+import os
+import xbmc
+import xbmcvfs
+import xbmcgui
+import xbmcvfs
+
+from json_rpc import json_rpc
+from simple_logging import SimpleLogging
+from translation import string_load
+
+log = SimpleLogging(__name__)
+ver = xbmc.getInfoLabel('System.BuildVersion')[:2]
+
+def walk_path(root_path, relative_path, all_files):
+    files = xbmcvfs.listdir(root_path)
+    found_paths = files[0]
+    found_files = files[1]
+
+    for item in found_files:
+        rel_path = os.path.join(relative_path, item)
+        all_files.append(rel_path)
+
+    for item in found_paths:
+        new_path = os.path.join(root_path, item)
+        rel_path = os.path.join(relative_path, item)
+        all_files.append(rel_path)
+        walk_path(new_path, rel_path, all_files)
+
+
+def get_value(name):
+    result = json_rpc('Settings.getSettingValue').execute({'setting': name})
+    return result['result']['value']
+
+
+def set_value(name, value):
+    params = {
+        'setting': name,
+        'value': value
+    }
+    result = json_rpc('Settings.setSettingValue').execute(params)
+    log.debug("Save Setting : {0} : {1} : {2} : {3}", name, value, type(value), result)
+
+
+def clone_skin():
```

Comment:
I don't think addons are allowed to do this. Have to check with others.

Reply:
are you referring to the clone skin function?

Reply:
have removed this now